components/timeline/TimelineHomeFilter.vue

```diff
@@ -0,0 +1,28 @@
+<script setup lang="ts">
+const homeFilter = useHomeFilter()
+
+const isActive = $computed(() => {
+  return !homeFilter.value.bot
+      || !homeFilter.value.sensitive
+      || !homeFilter.value.repost
+      || !homeFilter.value.mutual
+      || !homeFilter.value.tag
+})
+</script>
+
+<template>
+  <VDropdown>
+    <button btn-text>
+      <div v-if="!isActive" i-ri:filter-2-line />
+      <div v-else i-ri:filter-2-fill />
```

Comment:
Is it possible to conditionally swap icons in a better way?

Reply:
Cannot use `:class="isActive ? 'i-ri:filter-2-libe' : 'i-ri:filter-2-line'"`

Reply:
Ah 🤦
I forgot that these are just css classes
This is my first frontend js code I'm touching for one month... work (employer) sucks 😓